cypress/support/step_definition/commonCartridge/commonCartridgeSteps.spec.js

```diff
@@ -0,0 +1,85 @@
+import { BeforeAll, Given, Then } from "@badeball/cypress-cucumber-preprocessor";
+
+const fileName = "CC_Test_Kurs.imscc";
+
+BeforeAll(function() {
+	this.orgs = {}
+})
+
+Then("I have a file exported with pattern {string} and rename it", function (pattern) {
+	cy.verifyFile(pattern, fileName, 15000);
+});
+
+Given("the exported file is an archive and extracted", function () {
+	cy.task("commonCartridgeFileIsArchive", {
+		directory: Cypress.config("downloadsFolder"),
+		file: fileName,
+	});
+});
+
+Given("extracted content and files are deleted", function () {
+	cy.task("commonCartridgeCleanUp", {
+		directory: Cypress.config("downloadsFolder"),
+		file: fileName,
+	});
+});
+
+Then(
+	"a manifest exists in the common cartridge file with version {string} and title {string}",
```

Comment:
Please create a method in pageCourse.js and call it here in the step definition.